docker/start_supervisor

```diff
@@ -1,17 +0,0 @@
-#!/bin/bash
-
-#Make sure environment variables are set
-export GADGETRON_SUPERVISOR_NODAEMON=${GADGETRON_SUPERVISOR_NODAEMON:-"true"}
-export GADGETRON_RELAY_HOST=${GADGETRON_RELAY_HOST:-"127.0.0.1"}
-export GADGETRON_LOG_FILENAME=${GADGETRON_LOG_FILENAME:-"/tmp/gadgetron.log"}
-export OMP_THREAD_LIMIT=$(($(nproc)*2))
-#export OMP_NUM_THREADS=$(nproc)
-export OMP_WAIT_POLICY=passive
-#export OMP_DYNAMIC=false
-#export OMP_PROC_BIND=true
-export OPENBLAS_NUM_THREADS=1
```

Comment:
Should we be setting these variables as defaults in the container instead or are they not necessary?

Reply:
I thought about setting them in the `entrypoint.sh` but it is better (more transparent) to just use environment variables set in docker run (or whichever system you use). So like:

```
docker run -p 9002:9002 -e OMP_WAIT_POLICY=PASSIVE  ...
```